spyder/app/tour.py

```diff
@@ -144,68 +144,74 @@ def get_tour(index):
-                           "execute the code you wrote on the Editor. To do it "
-                           "you need to press the <b>F5</b> key.<br><br>"
-                           "This console comes with several "
+             {'title': _("IPython Console"),
+              'content': _("This is where you can run Python code, either "
+                           "from the Editor or interactively. To run the "
+                           "current file, press <b>F5</b> by default, "
+                           "or press <b>Ctrl+Enter</b> "
+                           "(<b>Cmd-Return</b> on Mac) to execute the current "
+                           "line or selection.<br><br>"
+                           "This console comes with many  "
                            "useful features that greatly improve your "
-                           "programming workflow (like syntax highlighting and "
-                           "inline plots). If you want to know more about them, "
-                           "please follow this <a href=\"{qtconsole_link}\">link</a>.<br><br>"
-                           "{button_text}").format(
-                           qtconsole_link=qtconsole_link, button_text=button_text),
+                           "programming workflow, like syntax highlighting, "
+                           "autocompletion, plotting and 'magic' commands. "
+                           "To learn more, check out the "
+                           f"<a href=\"{qtconsole_link}\">documentation</a>."
+                           f"<br><br>{button_text}"),
               'widgets': [sw.ipython_console],
-              'run': ["li = list(range(100))", "d = {'a': 1, 'b': 2}"]
+              'run': [
+                  "test_list_tour = list(range(100))",
+                  "test_dict_tour = {'a': 1, 'b': 2}",
+                  ]
               },
 
-             {'title': _("The Variable Explorer"),
+             {'title': _("Variable Explorer"),
               'content': _("In this pane you can view and edit the variables "
                            "generated during the execution of a program, or "
-                           "those entered directly in one of Spyder "
-                           "consoles.<br><br>"
-                           "As you can see, the Variable Explorer is showing "
-                           "the variables generated during the last step of "
-                           "this tour. By doing a double-click on any "
-                           "of them, a new window will be opened, where you "
+                           "those entered directly in the Console.<br><br>"
+                           "If you ran the code in the previous step, "
+                           "the Variable Explorer will show "
+                           "the list and dictionary objects it generated. "
+                           "By double-clicking any variable, "
+                           "a new window will be opened where you "

```

Comment:
Does this not interfere with the tour by creating strange focus problems? Just checking with you because I haven't tested that.

Reply:
The tour completely disables user interaction with Spyder on all the panes and menus, as it always has, so the user attempting to do this during the tour itself won't cause problems (it just won't work). Since users won't be able to actually do anything we tell users on any of the panes during the tour itself as opposed to after, and this was here before, I suggest we just leave it, otherwise there isn't much we can really say on any of them.